host/sgxload.c

```diff
@@ -737,7 +737,7 @@ oe_result_t oe_sgx_initialize_enclave(
                 &info.EInitToken,
                 sizeof(info.EInitToken),
                 (void*)&launch_token,
-                sizeof(launch_token)));
+                sizeof(info.EInitToken)));
```

Comment:
Hm what do you mean? The original code would error and prevent you from initializing enclaves since it was copying `sizeof(launch_token)`, but the static assert only checked if `sizeof(info.EnitToken) <= sizeof(launch_token)`. So, you only want to copy `sizeof(info.EInitToken)`
from `launch_token`, which was my change.